src/components/Modal/NewIssueModal/index.tsx

```diff
@@ -61,28 +61,50 @@ ${nextSteps}`,
       <div className={styles.container}>
         <div className={styles.halfLeft}>
           <div className={styles.section}>
-            <h5>Title</h5>
+            <h5 style={{ marginBottom: "8px" }}>Title</h5>
             <TextField textValue={title} onChangeHandler={setTitle} />
           </div>
 
           <div className={styles.userStory}>
             <h5 className={styles.userStoryHeading}>User story</h5>
             <p>As a</p>
-            <TextField textValue={role} onChangeHandler={setRole} />
+            <TextField
+              textValue={role}
+              onChangeHandler={setRole}
+              label="role"
+            />
             <p>I want to</p>
-            <TextField textValue={action} onChangeHandler={setAction} />
+            <TextField
+              textValue={action}
+              onChangeHandler={setAction}
+              label="action"
+            />
             <p>So that</p>
-            <TextField textValue={rationale} onChangeHandler={setRationale} />
+            <TextField
+              textValue={rationale}
+              onChangeHandler={setRationale}
+              label="rationale"
+            />
           </div>
 
           <div className={styles.section}>
-            <h5>What could be the suspected root cause?</h5>
-            <TextArea textValue={rootCause} onChangeHandler={setRootCause} />
+            <TextArea
+              textValue={rootCause}
+              onChangeHandler={setRootCause}
+              label={
+                topic?.type == "bugReport"
+                  ? "What could be the suspected root cause?"
+                  : "What value would be added by this feature?"
```

Comment:
Loving this change - would be good to update the `body` `useMemo()` up the top to include this too - perhaps this message could be made a `const` up the top & used both here and in the `body` string literal

Reply:
Oh shoot yep that makes sense, will do